lib/tarantool_server.py

```diff
@@ -481,6 +486,10 @@ def seek_wait(self, msg, proc=None, name=None, deadline=None):
                 if re.findall(msg, log_str):
                     return True
                 cur_pos = f.tell()
+
+        color_stdout("\nFailed to find '{}' pattern in {} "
+                     "logfile\n".format(msg, self.path), schema='error')
```

Comment:
I would also show some timing information: how much time we spent in trying to find the pattern.